sklearn/tests/test_calibration.py

```diff
@@ -540,7 +554,7 @@ def test_calibration_dict_pipeline(dict_data, dict_data_pipeline):
     """
     X, y = dict_data
     clf = dict_data_pipeline
-    calib_clf = CalibratedClassifierCV(clf, cv="prefit")
+    calib_clf = CalibratedClassifierCV(FrozenEstimator(clf), cv=2)
```

Comment:
This change highlights another small usability regression: it's difficult to understand why we would need to adjust the CV strategy when calibrating a frozen estimator on a dataset with less than 5 data-points per class. One could expect that cross-fitting would be entirely disabled in that case.

Again, I can live with it.

Reply:
I'd agree with you, but in reality, people with 5 data points shouldn't be doing statistical ML 😁

Reply:
In particular, meaningful calibration with 5 data points is quite challenging. Still, it reveals a rough edge in my opinion. But I believe we can fix it later if needed.